src/bin/rgb21.rs

```diff
@@ -47,12 +47,13 @@ pub enum Command {
         #[clap(short, long, default_value = "8")]
         precision: u8,
 
-        /// Asset parent ID
-        parent_id: Option<AsciiString>,
-
         /// Asset allocations, in form of <amount>@<txid>:<vout>
         allocations: Vec<OutpointValue>,
 
+        /// Asset parent ID
```

Comment:
Is there any reason to move the `parent_id` parameter after `allocations`?

Reply:
I changed it for usability reasons. 

This order of the parameters is similar to the order of the RGB20 lib. Nothing important =)